sdk/mixedreality/ci.yml

```diff
@@ -0,0 +1,31 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/mixedreality/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/mixedreality/
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
+  parameters:
+    ServiceDirectory: mixedreality
+    ArtifactName: packages
+    TestPipeline: true
```

Comment:
Is it correct that TestPipeline should be populated here? I know this was recently added to the Template, but I'm not sure if it is meant to be copied forward. /cc @chidozieononiwu @weshaggard

Reply:
I'd remove ```TestPipeline: true``` for now.

Reply:
Done. This did come from the template, so perhaps it should be removed from the template until it's recommended.